components/private/Availability/src/MDCAvailability.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MDC_AVAILABILITY
```

Comment:
This is a great idea, but I think it should be submitted separately from the Large Content Viewer work in this PR. Macros are also *very* tricky to get *just right* so it's probably worth reviewing and discussing independently.

Please split the MDCAvailability out to another PR. If/when it passes, we can take a stab at updating this code.  For now, please stick with our current boilerplate, 

```objc
#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
```

Reply:
Will break out into separate PR!